src/main/resources/application.conf

```diff
@@ -38,6 +38,8 @@ mantis {
 
   network {
     # Ethereum protocol version
+    # Supported versions:
+    # 63, 64 (experimental version which enables usage of messages with checkpointing information. In the future after ETCM-355, ETCM-356, it will be 66 probably)
     protocol-version = 63
```

Comment:
One last request for this PR, could you set protocol-version to 64 to the `testnet-internal-nomad.conf`?

It would be compliant with the fact that that testnet already has ECIP 1097 enabled